torchtune/utils/_distributed.py

```diff
@@ -242,6 +250,79 @@ def lora_wrap_fsdp(module: nn.Module, recurse: bool, **kwargs):
     return lora_wrap_fsdp
 
 
+def load_from_full_model_state_dict(
```

Comment:
This is a bit unfortunate but we have two different meanings of "full model" IIUC:
- One is related to FSDP i.e. full vs sharded - is that right?
- Other is LoRA i.e. full model vs LoRA adapters

The current function is a bit confusing since we pass in the adapter state_dict and this is referred to as ```full_sd``` in the function itself.

Reply:
good catch. you are right "full" is opposing to "sharded" here. What do you think if rename to "local"?  `load_from_local_model_state_dict` ?

Reply:
Can we add docstrings for the functions added in this file?